### ISSUE TITLE
Possible values for tokenFormat

### ISSUE THREAD

**Issue by jkarttunen** [d01000100/figma-token-engine#46]
Could not find this from docs.

```
 "tokenFormat": "TokenStudio",
```

It looks like this can have other values too (for Figma tokens), but could not find it from docs.

**Comment by d01000100** [comment]
Yeah, the docs definitely need more detailed work. The possible values so far are "TokensStudio" and "FigmaStyles"
